src/synapse/Player.php

```diff
@@ -278,8 +278,10 @@ public function dataPacket(DataPacket $packet, $needACK = false){
 			$pk->sendTo = $this->uuid;
 			$pk->type = $packet->type;
 			foreach($packet->entries as $entry){
-				array_pop($entry);
-				array_pop($entry);
+				if($packet->type !== PlayerListPacket::TYPE_REMOVE){
```

Comment:
There is error, because entry has 2 unwanted parametrs, but only in TYPE_ADD, I added check of type to fix it.